index.js

```diff
@@ -40,7 +53,11 @@ export const buildList = () => {
       if (s.includes('bg-white')) return
       if (s.includes('bg-transparent')) return
       if (s.includes('bg-none')) return
-      if (filterIgnoreList.includes(s.replace('.', ''))) return
+      if (s.includes('border-white')) return // Not used in Finn repos
```

Comment:
`border-white` is also used in FINN repos, so maybe this comment should be different?

http://finn-codesearch.svc.prod.finn.no/search?q=border-white&num=50